hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/SerializationUtilTest.java

```diff
@@ -102,6 +108,21 @@ public void testReadWriteNullableBoolean_whenInvalid()
         deserialize(bytes);
     }
 
+    @Test
+    public void testGetInterfacesCalledVerified() {
+
+        SerializationUtil serializationUtil = mock(SerializationUtil.class);
+
+       // HashSet implements Set<E>, Cloneable, java.io.Serializable, hence using it for this test
+        Class type = new HashSet().getClass();
+        final Set<Class> interfaces = new LinkedHashSet<Class>(5);
+
+        serializationUtil.getInterfaces(type, interfaces);
+
+        verify(serializationUtil, times(1)).getInterfaces(type, interfaces);
+
```

Comment:
minor: unnecessary new line

Reply:
Thank you for the suggestion, I have updated my pull request with a new commit to fix this.